### ISSUE TITLE
Raptor Owl Camera 640 on Ubuntu 20.04

### ISSUE THREAD

**Issue by leanedonze** [micro-manager/micro-manager#1728]
Hi there,

Thanks a lot for this open-source software.

I managed to successfully install Micro-Manager on my Jetson Orin Nano running Ubuntu 20.04.

Following the instructions from the user manual of my Raptor Owl Camera 640, I tried to select my device from the list to create a config file.
![image](https://github.com/micro-manager/micro-manager/assets/48278352/09907463-8fdd-4e3c-a6b7-62e4c0b9d240)

However, my device does not appear on the list. From https://micro-manager.org/RaptorEPIX, I believe this is because only Windows is supported as Operating System. Is there anything I can do about that?

I am mainly interested in Pycro Manager to have a Python API to interact with my camera, not really about the GUI interface of Micro-Manager. I guess the device will not be available either through Pycro Manager?

Thanks for your help

**Comment by nicost** [comment]
Yes, it sure looks like Raptor cameras are not supported under Linux.   Your best chance is to contact the vendor and tell them you would love to have Linux support.  When they hear this from a few people, they may well put the resources into making this happen.
